src/audit-logs/serializers/create-audit-log-schema.serializer.ts

```diff
@@ -0,0 +1,39 @@
+import { AuditLogSchema, CreateAuditLogSchemaResponse } from '../interfaces';
+
+function deserializeMetadata(metadata: {
+  properties?: Record<string, { type: string }>;
```

Comment:
```suggestion
  properties?: Record<string, { type: string | number | boolean, nullable: boolean }>;
```

Might be worth extracting this type also.